fastlane/Fastfile

```diff
@@ -958,11 +1000,11 @@ REPOSITORY_NAME="WordPress-Android"
 #####################################################################################
 # Private lanes
 #####################################################################################
-  private_lane :delete_old_changelogs do | options |
-    Dir.glob("./metadata/android/*/").each do | folder |
-      Dir["#{folder}changelogs/*"].each do | file |
-        File.delete(file) if Integer(File.basename(file, ".*")) < Integer(options[:build]) rescue puts "Cannot delete file #{file}"
-      end
+  private_lane :delete_old_changelogs do |options|
+    app = options[:app] || 'wordpress'
```

Comment:
I personally much prefer transparency because I find that minor verbosity helps with reading the code. (not this code, but the one that'll call this lane) Can we force the caller to specify the app for that benefit? Since it's ruby, we don't get the static typing, but still, at least it's not going to do something unexpected when I forget to specify a parameter.

What do you think?

Reply:
Yeah actually I think fastlane has a build-in `fetch_options` function (or is it a `fetch` instance method on `options`? Or a slightly different method name?) that can take care of it all: possibility to declare a default value (which I agree we should not), or to use an interactive prompt to ask for the value if none is provided (which is probably the better option)

Reply:
Since this is a private lane, I think we can just drop the default value and call it done. If I am understanding the setup correctly, we should always have access to the `options[:app]`, so if it's not passed to this lane, that means we just made a mistake, right?